packages/control-property-editor/src/panels/outline/Tree.tsx

```diff
@@ -75,7 +75,7 @@ export const Tree = (): ReactElement => {
                 // make sure that tree is fully rendered
                 const rect = node.getBoundingClientRect();
                 const outlineContainer = document.getElementsByClassName('section--scrollable')[0];
-                if (rect.top <= 20 || rect.bottom >= outlineContainer?.clientHeight) {
+                if (rect.top <= 75 || rect.bottom >= outlineContainer?.clientHeight) {
```

Comment:
can you please define variable for  `75` to give some semantic for `75`.

Reply:
Yes sure